src/coprocessor/codec/batch/lazy_column_vec.rs

```diff
@@ -113,6 +131,15 @@ impl LazyBatchColumnVec {
         self.columns[0].len()
     }
 
+    /// Debug asserts that all columns have equal length.
+    #[inline]
+    pub fn debug_assert_columns_equal_length(&self) {
+        let len = self.rows_len();
```

Comment:
Could we check and return directly here when it is not in debug mode?

Reply:
Good idea. Changed!